docs/fiddles/features/progress-bar/main.js

```diff
@@ -10,26 +10,18 @@ function createWindow () {
 
   win.loadFile('index.html')
 
-  const INCREMENT = 0.02
+  const INCREMENT = 0.03
   const INTERVAL_DELAY = 100 // ms
-  const INDETERMINATE_PAUSE = 5000 // ms
 
   let c = 0
   progressInterval = setInterval(() => {
     // update progress bar to next value
+    // values between 0 and 1 will show progress, >1 will show indeterminate or stick at 100%
     win.setProgressBar(c)
-
-    if (c > 1) {
-      // set to indeterminate state
-      win.setProgressBar(2)
-
-      // stop the interval
-      clearInterval(progressInterval)
-
-      // and after INDETERMINATE_PAUSE reset the progress bar
-      indeterminateTimeout = setTimeout(() => win.setProgressBar(-1), INDETERMINATE_PAUSE)
-    }
-    c += INCREMENT
+    
+    // increment or reset progress bar
+    if (c < 2) c += INCREMENT
+    else c = 0
```

Comment:
maybe do this
```suggestion
    else c = (-INCREMENT * 5)
```
so that it goes away for a bit, too?

I'm not sure about single-line code blocks (@erickzhao can comment on the style guidance here) -  I don't like them, but that's just a personal thing.

Reply:
Style-wise, I think one-liner returns are fine but would rather make the code more readable for newbies

Reply:
Done in `432d`.